src/less/split-button/base/split-button.less

```diff
@@ -0,0 +1,18 @@
+@import "../../mixins/utility/utility-mixins.less";
+
+button.btn.split-button {
+    border-radius: 24px 0 0 24px;
+}
+
+.menu-button button.split-button {
```

Comment:
Does split button work without menu-button? If someone wanted to create a split button on their own without a dependency on menu-button would it work?